.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Please ensure that the following information is filled before filing
+--------------------------------------------------------------------
+
+Please note that the alot version should be either the latest released version,
```

Comment:
I would phrase it like this: "Please try to verify that the issue exists in the latest release or, even better, the current master."